terra-batch-libchart/templates/_reverse-proxy.tpl

```diff
@@ -82,34 +82,34 @@ data:
       server {
         listen 8000;
 
-        # For now, keep serving 'cromwell' APIs at proxy root, but we should consider this deprecated and
-        # update clients to use `/cromwell/` when talking to the Cromwell APIs.
-        location /api/ {
-          proxy_pass http://{{ include "app.fullname" . }}-cromwell-svc:8000/api/;
-        }
-        location /engine/ {
-          proxy_pass http://{{ include "app.fullname" . }}-cromwell-svc:8000/engine/;
-        }
-
         # Reference to ngingx's local content
         location /index.html {
           alias /www/data/index.html;
         }
 
         # Proxying to other hosts by subpath:
-        location /cbas/ {
-          proxy_pass http://{{ include "app.fullname" . }}-cbas-svc:8080/;
-        }
-        location /cromwell/ {
-          proxy_pass http://{{ include "app.fullname" . }}-cromwell-svc:8000/;
+        if ({ .Values.cromwell.coaEnabled } = 'true') {
```

Comment:
This gates everythong on `.Values.cromwell.coaEnabled` -- think we want to change cbas/wds/cromwell for each `location`?

Reply:
I updated the logic to surround each `location` -- let me know if that's what you had in mind!